### ISSUE TITLE
Feature Request: The Java DataTable class should support re-setting column labels and changing cell values

### ISSUE THREAD

**Issue by orwant** [orwant/google-visualization-issues#68]
Original [issue 70](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=70) created by orwant on 2009-09-30T16:20:18.000Z:

<b>What would you like to see us add to this API?</b>
The DataTable class supports adding rows and column descriptions.  However
once the rows or columns are added, you cannot change a column label or
change the value in a single row cell.  This would be very useful for cases
where you want to apply some post-processing to a particular column (i.e.
change an ID column to an URL).  The only way to do this right now is to
basically rebuild the entire DataTable, and changing the cells or column
labels in the new DataTable.

<b>What component is this issue related to (PieChart, LineChart, DataTable,</b>
<b>Query, etc)?</b>
The Java DataTable class

<b>*********************************************************</b>
<b>For developers viewing this issue: please click the 'star' icon to be</b>
<b>notified of future changes, and to let us know how many of you are</b>
<b>interested in seeing it resolved.</b>
<b>*********************************************************</b>

**Comment by orwant** [comment]
Comment [#1](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=70#c1) originally posted by orwant on 2009-10-19T09:30:31.000Z:

are you referring to the DataTable class on the data source server or in the GWT client?

**Comment by orwant** [comment]
Comment [#2](https://code.google.com/p/google-visualization-api-issues/issues/detail?id=70#c2) originally posted by orwant on 2009-10-21T18:57:08.000Z:

the GWT client
